src/odemis/driver/test/xt_client_test.py

```diff
@@ -106,7 +106,7 @@ def test_move_stage(self):
 
         stage_position = self.stage.position.value.copy()
         # test move_stage method actually moves HW by the requested value
-        self.assertAlmostEqual(stage_position['x'], relative_position['x'] + new_pos['x'])
+        self.assertAlmostEqual(stage_position['x'], relative_position['x'] - new_pos['x'])
```

Comment:
That's not right, is it? It seems an unrelated change, concerning the "inverted" issue. In any case, I'm quite convince the original code was fine (it should be a + everywhere).